src/styles/main.scss

```diff
@@ -189,10 +189,10 @@ div.article-container{
   }
 
   .external-link-tag-wrapper{
-	display: inline-block;
-	.external-link-icon {
-	  margin: 0 0 0 3px;
-	}
+	  display: inline;
```

Comment:
formatting is off?

Reply:
I fixed the formatting. It was off to begin with it looks good now
<img width="313" alt="Screen Shot 2022-03-15 at 2 27 24 PM" src="https://user-images.githubusercontent.com/86666932/158446625-b52ecca4-7cec-4288-8ad9-4f83c781ccf2.png">